main/appserver/packager/glassfish-jcdi/pom.xml

```diff
@@ -95,6 +95,10 @@
             <artifactId>weld-osgi-bundle</artifactId>
         </dependency> 
         <dependency>
+            <groupId>org.jboss.classfilewriter</groupId>
```

Comment:
The jboss-classfilewriter dependency, which is used by Weld for runtime bytecode generation, is no longer bundled within the Weld OSGi bundle. Instead, this dependency needs to be deployed separately to the OSGi container.
This change was introduced in Weld 2.3.